src/views/signup/components/survey/SurveyOneComponent.vue

```diff
@@ -94,37 +94,34 @@
 import { reactive, ref } from 'vue';
 
 import AlertModal from '@/components/AlertModal.vue';
-
-import { INITIAL_QUESTION_LIST } from '../../constants/question';
+import { SURVEY_LIST } from '@/views/signup/constants/survey';
 
 const emit = defineEmits(['next']);
 
+const surveyList = SURVEY_LIST.filter(question => question.id <= 3);
+
 // 답변
-const answers = reactive({
-  age: null,
-  income: null,
-  save: null,
-  habit: null,
-});
+const answers = reactive(
+  surveyList.map(question => ({
+    [question.id]: null,
+  }))
+);
 
 // 모달 상태
 const showModal = ref(false);
 
 // 응답 선택
-function selectOption(questionId, value) {
+const selectOption = (questionId, value) => {
   answers[questionId] = value;
-}
+};
 
 // 다음 버튼 클릭 시
 function handleNext() {
   //answers의 value들을 배열에 담아 값이 null이 아닌지(모든 항목이 선택되었는지) 확인
-  const isAllAnswered = Object.values(answers).every(value => value !== null);
-  if (!isAllAnswered) {
+  if (answers.some(answer => answer === null)) {
```

Comment:
```suggestion
if (Object.values(answers).some(answer => answer === null)) {
```
찾아보니 원래 'some'은 배열에서만 동작한다고 하네요! 지금도 동작해서 상관은 없지만 정석대로 한다면 이런식으로 바꿔보셔도 좋을 것 같습니당

Reply:
위에서 선언한 값을 배열로 선언해서 some 함수를 사용했습니다!
객체로 초기화를 한다면 제안해주신 방법대로 하면 좋을 거 같습니다!

여러 값들에 접근하고 다루는데에는 객체보다는 배열이 더 적합하고 직관적이라고 생각이 들어서 배열로 선언했습니다!